set.mm

```diff
@@ -319161,6 +319161,52 @@ can be the countable union of rare closed subsets (where rare means
       HZJSDGHZDKHQRSLQRTSABCDEFMNDOP $.
   $}
 
+  ${
+    cssbn.x $e |- X = ( W |`s U ) $.
+    cssbn.s $e |- S = ( LSubSp ` W ) $.
+    cssbn.d $e |- D = ( ( dist ` W ) |` ( U X. U ) ) $.
+    $( Banach space property of a closed subspace: a closed subspace of a
+       normed vector space with a closed scalar field is a Banach space.
```

Comment:
You can remove the part before the colon, and begin at "A closed subspace..."
also: closed scalar field --> complete scalar field
And remove the remark. There are not many compact linear subspaces... maybe "complete" was meant, but anyway the remark is out of place.

Reply:
Oh, I see... that remark is a kind of explanation why the statement of the theorem differs from its comment... Let's find a better wording, then...

Reply:
Do you think it would be doable to formalize:
```
   cssbn $p |- ( ( W e. NrmVec /\ ( X e. CMetSp /\ U e. S ) ) -> X e. Ban ) $=
```
?

Also: the juggling between U and X is annoying. One should define LSubSp using the structure restriction df-ress...